package.json

```diff
@@ -27,7 +27,7 @@
     "storybook": "start-storybook -p 6006",
     "test": "yarn test:unit && yarn test:a11y",
     "posttest": "yarn lint",
-    "test-coverage": "yarn cmsds test ./packages --collectCoverage",
+    "test:coverage": "yarn test:unit --collectCoverage",
```

Comment:
Nice 👍. Maybe it can be even more specific, like `test:unit:coverage`

Reply:
Good call